app/src/main/java/ru/artofmainstreams/androidacademy2020/FragmentMoviesList.kt

```diff
@@ -0,0 +1,42 @@
+package ru.artofmainstreams.androidacademy2020
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.material.card.MaterialCardView
+
+/**
+ * Фрагмент со списком фильмов
+ *
+ * @author Andrei Khromov on 08.01.2021
+ */
+class FragmentMoviesList : Fragment() {
+    private var clickListener: OnClickListener? = null
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        if (context is OnClickListener) {
+            clickListener = context
+        }
```

Comment:
Котлин позволяет писать лаконичнее
```suggestion
        clickListener = context as? OnClickListener
```